crm_won_restrict_per_stage/views/crm_views.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="crm_stage_form" model="ir.ui.view">
+        <field name="name">crm.stage.form.inherit</field>
+        <field name="model">crm.stage</field>
```

Comment:
Don't mix in the same file views from different models. That's an annoying habit in Odoo basecode that we try to avoid here in OCA.